### PR TITLE
Morgue and Crematorium Repositions

### PR DESCRIPTION
**Pull request by Chuckleberry-Finn**
[Changelogs]: 
:cl: Jittai
tweak: Some of the Morgue Trays and Crematoriums on Box, Delta, and Meta have been re-positioned to make use of their new directional states.
/:cl:

Very minor re-positions, some are just turned, adds no extra trays.

Box:
 

Meta:

Delta:
Note: Moved a random-event-spawner up 1 tile and some of the flooring changed to match pattern.

Chapel Backroom Respectively:
Box Meta Delta
box meta delta

[why]: To make use of the new directional states, gives a bit more space in the rooms, and arguably looks better.

redo of: #35110

### CODE REVIEW

**Comment by Armhulen** [comment]
I don't like th cremator change but the rest is great

**Comment by ChuckTheSheep** [comment]
I tweaked them a bit for variety's sake.

**Comment by DaxDupont** [comment]
Delta morgue feels too big now

**Comment by ChuckTheSheep** [comment]
It is the power of feng shui.

File: _maps/map_files/BoxStation/BoxStation.dmm [hunk 1]
```diff
@@ -23075,6 +23075,10 @@
 /area/medical/morgue)
 "bho" = (
 /obj/machinery/atmospherics/pipe/simple/supply/hidden,
+/obj/structure/bodycontainer/morgue{
+	icon_state = "morgue1";
```
Comment (ShizCalev) [inline _maps/map_files/BoxStation/BoxStation.dmm]: bad

File: _maps/map_files/Deltastation/DeltaStation2.dmm [hunk 2]
```diff
@@ -90597,18 +90597,27 @@
 	},
 /area/medical/morgue)
 "dDM" = (
-/obj/structure/bodycontainer/morgue,
+/obj/structure/bodycontainer/morgue{
+	icon_state = "morgue1";
```
Comment (ShizCalev) [inline _maps/map_files/Deltastation/DeltaStation2.dmm]: remove

File: _maps/map_files/BoxStation/BoxStation.dmm (hunk 1 above)
Comment (ShizCalev) [inline _maps/map_files/BoxStation/BoxStation.dmm]: remove

File: _maps/map_files/Deltastation/DeltaStation2.dmm [hunk 3]
```diff
@@ -90597,18 +90597,27 @@
...
+	dir = 2
+	},
 /turf/open/floor/plating,
 /area/medical/morgue)
 "dDN" = (
-/obj/structure/bodycontainer/morgue,
+/obj/structure/bodycontainer/morgue{
+	icon_state = "morgue1";
```
Comment (ShizCalev) [inline _maps/map_files/Deltastation/DeltaStation2.dmm]: remove

File: _maps/map_files/Deltastation/DeltaStation2.dmm [hunk 4]
```diff
@@ -90597,18 +90597,27 @@
...
+	},
 /turf/open/floor/plasteel/vault{
 	dir = 8
 	},
 /area/medical/morgue)
 "dDO" = (
-/obj/structure/bodycontainer/morgue,
 /obj/effect/decal/cleanable/dirt,
+/obj/structure/bodycontainer/morgue{
+	icon_state = "morgue1";
```
Comment (ShizCalev) [inline _maps/map_files/Deltastation/DeltaStation2.dmm]: remove

File: _maps/map_files/Deltastation/DeltaStation2.dmm [hunk 5]
```diff
@@ -93091,6 +93105,10 @@
 "dIW" = (
 /obj/effect/decal/cleanable/dirt,
 /obj/machinery/light/small,
+/obj/structure/bodycontainer/morgue{
+	icon_state = "morgue1";
```
Comment (ShizCalev) [inline _maps/map_files/Deltastation/DeltaStation2.dmm]: remove

File: _maps/map_files/Deltastation/DeltaStation2.dmm [hunk 6]
```diff
@@ -93105,6 +93123,10 @@
 /obj/structure/sign/poster/official/bless_this_spess{
 	pixel_y = -32
 	},
+/obj/structure/bodycontainer/morgue{
+	icon_state = "morgue1";
```
Comment (ShizCalev) [inline _maps/map_files/Deltastation/DeltaStation2.dmm]: remove

File: _maps/map_files/Deltastation/DeltaStation2.dmm [hunk 7]
```diff
@@ -93122,14 +93144,20 @@
 	},
 /area/medical/morgue)
 "dIZ" = (
-/obj/structure/bodycontainer/morgue,
 /obj/effect/decal/cleanable/dirt,
+/obj/structure/bodycontainer/morgue{
+	icon_state = "morgue1";
```
Comment (ShizCalev) [inline _maps/map_files/Deltastation/DeltaStation2.dmm]: remove

File: _maps/map_files/Deltastation/DeltaStation2.dmm [hunk 8]
```diff
@@ -93122,14 +93144,20 @@
...
+	dir = 1
+	},
 /turf/open/floor/plating,
 /area/medical/morgue)
 "dJa" = (
 /obj/effect/decal/cleanable/dirt,
 /obj/structure/light_construct/small,
-/obj/effect/landmark/event_spawn,
+/obj/structure/bodycontainer/morgue{
+	icon_state = "morgue1";
```
Comment (ShizCalev) [inline _maps/map_files/Deltastation/DeltaStation2.dmm]: remove

File: _maps/map_files/Deltastation/DeltaStation2.dmm [hunk 9]
```diff
@@ -100955,10 +100983,11 @@
 /turf/closed/wall/r_wall,
 /area/chapel/office)
 "dZO" = (
+/obj/machinery/atmospherics/pipe/simple/supply/hidden,
 /obj/structure/bodycontainer/crematorium{
-	id = "cremawheat"
+	icon_state = "crema1";
```
Comment (ShizCalev) [inline _maps/map_files/Deltastation/DeltaStation2.dmm]: remove
put the ID back in.

File: _maps/map_files/Deltastation/DeltaStation2.dmm [hunk 10]
```diff
@@ -105025,6 +105054,49 @@
 	},
 /turf/open/floor/plating,
 /area/maintenance/port)
+"QOn" = (
+/obj/effect/decal/cleanable/dirt,
+/obj/structure/bodycontainer/morgue{
+	icon_state = "morgue1";
```
Comment (ShizCalev) [inline _maps/map_files/Deltastation/DeltaStation2.dmm]: remove

File: _maps/map_files/Deltastation/DeltaStation2.dmm [hunk 11]
```diff
@@ -105025,6 +105054,49 @@
...
+	dir = 2
+	},
+/turf/open/floor/plasteel/vault{
+	dir = 5
+	},
+/area/medical/morgue)
+"QOo" = (
+/obj/effect/decal/cleanable/dirt,
+/obj/structure/bodycontainer/morgue{
+	icon_state = "morgue1";
```
Comment (ShizCalev) [inline _maps/map_files/Deltastation/DeltaStation2.dmm]: remove

File: _maps/map_files/Deltastation/DeltaStation2.dmm [hunk 12]
```diff
@@ -105025,6 +105054,49 @@
...
+	dir = 5
+	},
+/area/medical/morgue)
+"QOp" = (
+/obj/effect/decal/cleanable/dirt,
+/turf/open/floor/plasteel/neutral,
+/area/medical/morgue)
+"QOq" = (
+/obj/structure/bodycontainer/morgue{
+	icon_state = "morgue1";
```
Comment (ShizCalev) [inline _maps/map_files/Deltastation/DeltaStation2.dmm]: remove

File: _maps/map_files/Deltastation/DeltaStation2.dmm [hunk 13]
```diff
@@ -105025,6 +105054,49 @@
...
+	dir = 1
+	},
+/turf/open/floor/plasteel/vault{
+	dir = 8
+	},
+/area/medical/morgue)
+"QOr" = (
+/obj/effect/decal/cleanable/dirt,
+/obj/structure/bodycontainer/morgue{
+	icon_state = "morgue1";
```
Comment (ShizCalev) [inline _maps/map_files/Deltastation/DeltaStation2.dmm]: remove

File: _maps/map_files/MetaStation/MetaStation.dmm [hunk 14]
```diff
@@ -65518,6 +65518,10 @@
 /area/medical/medbay/aft)
 "cFU" = (
 /obj/machinery/light/small,
+/obj/structure/bodycontainer/morgue{
+	icon_state = "morgue1";
```
Comment (ShizCalev) [inline _maps/map_files/MetaStation/MetaStation.dmm]: remove

File: _maps/map_files/MetaStation/MetaStation.dmm [hunk 15]
```diff
@@ -68608,8 +68612,11 @@
 	},
 /area/medical/virology)
 "cLT" = (
-/obj/structure/bodycontainer/crematorium,
 /obj/effect/decal/cleanable/cobweb,
+/obj/structure/bodycontainer/morgue{
+	icon_state = "morgue1";
```
Comment (ShizCalev) [inline _maps/map_files/MetaStation/MetaStation.dmm]: remove

File: _maps/map_files/MetaStation/MetaStation.dmm [hunk 16]
```diff
@@ -75231,6 +75239,10 @@
 	dir = 2;
 	network = list("SS13","Medbay")
 	},
+/obj/structure/bodycontainer/morgue{
+	icon_state = "morgue1";
```
Comment (ShizCalev) [inline _maps/map_files/MetaStation/MetaStation.dmm]: remove

File: _maps/map_files/MetaStation/MetaStation.dmm [hunk 17]
```diff
@@ -80644,6 +80656,27 @@
...
+"QuY" = (
+/obj/structure/bodycontainer/morgue{
+	icon_state = "morgue1";
+	dir = 8
+	},
+/turf/open/floor/plasteel/dark,
+/area/medical/morgue)
+"QuZ" = (
+/obj/structure/bodycontainer/crematorium{
+	icon_state = "crema1";
```
Comment (ShizCalev) [inline _maps/map_files/MetaStation/MetaStation.dmm]: isn't the ID cremawheat?

File: _maps/map_files/MetaStation/MetaStation.dmm [hunk 18]
```diff
@@ -80644,6 +80656,27 @@
 /obj/effect/landmark/event_spawn,
 /turf/open/floor/plasteel/white,
 /area/science/circuit)
+"QuX" = (
+/obj/structure/bodycontainer/morgue{
+	icon_state = "morgue1";
```
Comment (ShizCalev) [inline _maps/map_files/MetaStation/MetaStation.dmm]: cremawheat

File: _maps/map_files/MetaStation/MetaStation.dmm [hunk 19]
```diff
@@ -80644,6 +80656,27 @@
...
+	dir = 8
+	},
+/turf/open/floor/plasteel/dark,
+/area/medical/morgue)
+"QuY" = (
+/obj/structure/bodycontainer/morgue{
+	icon_state = "morgue1";
```
Comment (ShizCalev) [inline _maps/map_files/MetaStation/MetaStation.dmm]: remove

**Review by ShizCalev — CHANGES_REQUESTED** [review]
do the things please.

**Comment by ChuckTheSheep** [comment]
@ShizCalev How do I remove the icon_state lines? And I don't think crematoriums need IDs unless there are multiple across the station- I wasn't aware of any of them using unique IDs.

**Comment by ShizCalev** [comment]
open the map in notepad++ and remove the line.

**Comment by ChuckTheSheep** [comment]
I was able to figure it out in editor, had to make the instances myself rather than auto-generate.

**Comment by ChuckTheSheep** [comment]
Should be good to go.

File: _maps/map_files/MetaStation/MetaStation.dmm [hunk 20]
```diff
@@ -69371,7 +69368,11 @@
 /turf/open/floor/plating/airless,
 /area/medical/virology)
 "cNt" = (
-/obj/structure/bodycontainer/morgue,
+/obj/machinery/button/crematorium{
+	id = "cremawheat";
+	pixel_x = -26;
+	req_access_txt = "27"
```
Comment (Son-of-Space) [inline _maps/map_files/MetaStation/MetaStation.dmm:69374]: caught almost 4 years later because the change didn't work.

**Pull request merged**